jsonnet/jsonnetfile.json

```diff
@@ -17,7 +17,7 @@
           "subdir": "jsonnet/kube-prometheus"
         }
       },
-      "version": "release-0.10"
```

Comment:
Change all jsonnet packages to its main/master branch.

Reply:
for the purpose of fixing this bug and reducing the change-set, perhaps it is better in this case to only make relevant changes, wdyt?